src/Microsoft.VisualStudio.Threading.Tests/AsyncLazyTests.cs

```diff
@@ -194,7 +194,7 @@ public void ValueFactoryThrowsSynchronously(bool specifyJtf)
             var task2 = lazy.GetValueAsync();
             Assert.Same(task1, task2);
             Assert.True(task1.IsFaulted);
-            Assert.IsType<ApplicationException>(task1.Exception.InnerException);
+            Assert.IsType<ApplicationException>(task1.Exception?.InnerException);
```

Comment:
I'd use a suppression for access of `Task.Exception` when we already see `Task.IsFaulted` is true.

```suggestion
            Assert.IsType<ApplicationException>(task1.Exception!.InnerException);
```

Reply:
Good point. Done.